js-codeshift/warn-to-log.batch.yml

```diff
@@ -0,0 +1,18 @@
+name: codeshift-demo
+description: Replace console.warn by console.log
```

Comment:
maybe it would be simpler to include the transformer script in here, so you don't need to rebuild the docker image each time you adjust the script:

```yaml
file: &jscodeshift_file
  /warn-to-log.ts: |
    export default function transformer(file, api) {
      const j = api.jscodeshift;
      const root = j(file.source);
      const consoleLogCalls = root.find(j.CallExpression, {
          callee: {
              object: {
                  name: 'console'
              },
              property: {
                  name: 'warn'
              }
          }
      });
      consoleLogCalls.forEach(p => {
          p.node.callee.property.name = 'log';
      });
      return root.toSource();
    };
name: codeshift-demo
description: Replace console.warn by console.log
# Repositories containing this package
on:
  - repositoriesMatchingQuery: console.warn
steps:
    - run: npx jscodeshift -t /warn-to-log.ts . --extensions js,ts --parser=ts
      files: *jscodeshift_file
      container: malomarrec/codeshift
changesetTemplate:
  title: Replace console.warn by console.log in .js files
  body: Because why not?
  branch: batch-changes/warn-to-log # Push the commit to this branch.
  commit:
    message: js-warn-to-log
  published: false
```

(untested)

Reply:
Fixed

Reply:
oh no 🤦